file/file-alfresco/src/main/java/uk/gov/justice/services/file/alfresco/requester/AlfrescoFileRequester.java

```diff
@@ -51,19 +51,23 @@
             } else {
                 throw new FileOperationException(format("Alfresco is unavailable with response status code: %d", responseStatus.getStatusCode()));
             }
-        } catch (ProcessingException processingException) {
+        } catch (final ProcessingException processingException) {
             throw new FileOperationException("Error fetching resource from Alfresco", processingException);
         }
     }
 
-    private String alfrescoUriOf(final String fieldId, final String fileName) {
+    private String alfrescoUriOf(final String fieldId, final String fileName, final boolean stream) {
         final StringBuilder requestBuilder = new StringBuilder();
         requestBuilder.append(alfrescoWorkspacePath);
         requestBuilder.append(fieldId);
         requestBuilder.append(URL_SEPARATOR);
         requestBuilder.append("content");
         requestBuilder.append(URL_SEPARATOR);
         requestBuilder.append(fileName);
+        // a means attach -> a = true means don't stream!
+        if (!stream) {
```

Comment:
When we send the reply do we not need to do something different as well?

For the request this may be the only modification required but I thought to return it we had to do something differently? maybe not? or maybe it is already implemented (just can't see on Github). Could you just double check please :) - PM me if you need more discussion.